Clarify install of docker

magnusoverli opened the pull request:
Updated description of installation to avoid possible confusion regarding docker as a dependency.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

magnusoverli committed:
Clarify install of docker

Updated description of installation to avoid possible confusion regarding docker as a dependency.

frenck commented:
@magnusoverli In general I approve, maybe we need to clarify a little bit more on this being a method/example on installing Ubuntu specifically?

magnusoverli committed:
Details on tested environment

Underlining Ubuntu as the tested environment for the given instructions.

magnusoverli committed:
Removed redundant docker install step

Removed step from hass.io installation instructions wherein installing docker. Installation of Docker-CE is already documented.

frenck committed:
:pencil2: Tweaks

frenck commented:
Thanks, @magnusoverli! 👍 
This helps a lot! 🏅

ludeeus commented on source/hassio/installation.markdown:
```diff
@@ -72,16 +73,30 @@ This is the list of packages you need to have available on your system that will
  - software-properties-common
 
-To perform the Hass.io installation, run the following command as root:
+You also need to have Docker-CE installed. There is well-documented procedures for installing Docker on Ubuntu at [Docker.com](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
```
This could confuse some users (to think that only ubuntu are supported)
```suggestion
You also need to have Docker-CE installed. There are well-documented procedures for installing Docker at [Docker.com](https://docs.docker.com/install/), you can find installation steps for your distro in the menu on the left.
```
I suggest changing the link to be more generic.

frenck replied:
agreed, correcting...
